examples/README.md

```diff
@@ -28,7 +28,7 @@ cargo build --package=examples --examples
 
 This module demonstrates how to create a minimal dynamic module with `http_request_handler`, that checks for User-Agent headers and returns status code 403 if UA starts with `curl`, if a module is disabled then uses `core::Status::NGX_DECLINED` to indicate the operation is rejected, for example, because it is disabled in the configuration (`curl off`). Additionally, it demonstrates how to write a defective parser.
 
-An example of nginx configuration file that uses that module can be found at [curl.conf](./curl.conf).
+An example of an Nginx configuration file that uses that module can be found at [curl.conf](./curl.conf).
```

Comment:
I haven't been checking these elsewhere, but I'm pretty sure the convention is to all-caps NGINX.

I'll confirm with Jodie.